.github/workflows/cd-docs.yml

```diff
@@ -2,9 +2,8 @@ name: deploy-docs
 on:
   workflow_dispatch:
   push:
-    # Uncomment these lines before merge
-    # branches:
-    #   - master
+    branches:
+      - master
```

Comment:
@peytondmurray I think we should allow this to run for push on any branch so developers can preview the deployment from their forks. If we make the reasonable assumption that nobody is pushing directly to upstream/master, then this will not affect the main docs deployment.

Reply:
Right now we have no way of deploying builds of individual branches - for now it is up to the users to build the docs and examine their changes locally, and for reviewers to pay due diligence in asking PR authors to provide screenshots/description of major changes to the docs.

Nobody will be pushing to `master`, but we still want this enabled so that when github itself makes a merge commit and pushes it to master the docs will be rebuilt. As it stands _every PR will force-push to `gh-pages`_.